app/bundles/FormBundle/EventListener/FormValidationSubscriber.php

```diff
@@ -100,4 +111,53 @@ private function fieldTelValidation(Events\ValidationEvent $event): void
             }
         }
     }
+
+    private function fieldCheckboxGroupValidation(Events\ValidationEvent $event): void
+    {
+        $field = $event->getField();
+        if ('checkboxgrp' !== $field->getType()) {
+            return;
+        }
+
+        $value       = $event->getValue();
+        $selectedCnt = 0;
+
+        if (is_array($value)) {
+            foreach ($value as $v) {
+                if ('' !== $v && null !== $v) {
+                    ++$selectedCnt;
+                }
+            }
+        } elseif ('' !== $value && null !== $value) {
+            $selectedCnt = 1;
+        }
```

Comment:
We could avoid repeating the condition if the value would be always array:

```suggestion
        if (!is_array($value)) {
            $value = [$value];
        }

        foreach ($value as $v) {
            if ('' !== $v && null !== $v) {
                ++$selectedCnt;
            }
        }
```